chainer/functions/noise/dropout.py

```diff
@@ -81,5 +94,6 @@ def dropout(x, ratio=.5, **kwargs):
     argument.assert_kwargs_empty(kwargs)
 
     if configuration.config.train:
-        return Dropout(ratio)(x)
+        y, = Dropout(ratio).apply((x,))
+        return y
```

Comment:
replace this line with `return Dropout(ratio).apply((x,))[0]` and remove L:97 is OK here, I think